### DISCUSSION TITLE
List countries, etc in relative order (by visitor growth)

### DISCUSSION THREAD

**Discussion by muesliq** [plausible/analytics#2097] [Anything Else]
It would be great to quickly be able to see dimensions not ranked absolutely but by their relativ performance, i.e. growth of unique visitors. I was looking for that information for **countries** but the same could apply to every other dimension, most notably **sources**, **pages** or **search terms**.

Relative growth is already shown in the chart area ("Unique visitors - up 10%") but there is no easy way to compare those values among each other. That could easily be solved by a switch from absolute numbers (visitors) in the list view to relative numbers (visitor growth).

<img width="536" alt="Screenshot 2022-08-11 at 17 26 38" src="https://user-images.githubusercontent.com/1606618/184170886-31eb1690-ff01-45a8-98a2-462855ac61d8.png">

Some measures would be needed to exclude irrelevant items like a 300% growth from 1 to 3 visitors. That could be handled by some kind of threshold, for example x percent of total visitors. Only items above that threshold would be listed. The threshold could be configurable via settings.

PS: Google Search Console Insights has a similar feature: Most trending queries.

(Sorry if this was discussed before. I did search but didn't find anything.)

**Comment by RobertJoonas** [comment]
Thanks @muesliq, for your feedback!

There's a similar feature request for showing growth for top pages: https://github.com/plausible/analytics/discussions/2099

I think this discussion falls more under the "feature request" category, so I've now changed it :slightly_smiling_face: 

This definitely sounds interesting and would be a great addition in the future. Unfortunately, I cannot make any timeline promises on when we'll be able to work on this. We're  small team with limited development resources and prioritize our work based on the number of requests.

Hope this makes sense, thanks!
